docs/faq.md

```diff
@@ -41,6 +42,42 @@ kubectl run jib-deployment --image=<image name>
 
 For more information, see [steps 4-6 of the Kubernetes Engine deployment tutorial](https://cloud.google.com/kubernetes-engine/docs/tutorials/hello-app#step_4_create_a_container_cluster).
 
+#### Where is bash?
+
+By default, Jib uses [`distroless/java`](https://github.com/GoogleContainerTools/distroless/tree/master/java) as the base image. Visit the [distroless project](https://github.com/GoogleContainerTools/distroless#distroless-docker-images) for more information about distroless images.
```

Comment:
I think it'd still be a good idea to mention that distroless doesn't have a shell here. ("By default, Jib uses `distroless/java` as the base image, which does not have a shell. Visit the...")